apis/apiextensions/v1/composition_transforms.go

```diff
@@ -110,6 +110,8 @@ type MatchTransform struct {
 	// The fallback value that should be returned by the transform if now pattern
 	// matches.
 	FallbackValue extv1.JSON `json:"fallbackValue,omitempty"`
+	// If set to true, the input value will be returned if no pattern matches.
```

Comment:
nit: should we mention that this is mutually exclusive with FallbackValue?

Reply:
An alternative API here would be to use an enum. For example:

```yaml
fallbackTo: Value
fallbackValue: whatever
```

```yaml
fallbackTo: Input
```

This fits with [Kubernetes API conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#primitive-types), which say:

> Think twice about bool fields. Many ideas start as boolean but eventually trend towards a small set of mutually exclusive options. Plan for future expansions by describing the policy options explicitly as a string type alias (e.g. TerminationMessagePolicy).

Reply:
Good idea. I've changed the PR to use that API